mlflow/openai/api_request_parallel_processor.py

```diff
@@ -316,7 +331,7 @@ def process_api_requests(
             f"{status_tracker.num_tasks_failed} tasks failed. See logs for details."
         )
     if status_tracker.num_rate_limit_errors > 0:
```

Comment:
Because of the new logic, we ideally don't need to keep count of `num_rate_limit_errors` anymore. The debugging logs below are not super helpful. Its a nit. So no need for immediate change